servers/src/common/types.rs

```diff
@@ -214,6 +217,7 @@ impl Default for ServerConfig {
 			dandelion_config: pool::DandelionConfig::default(),
 			stratum_mining_config: Some(StratumServerConfig::default()),
 			chain_type: ChainTypes::default(),
+			future_time_limit: 5 * 60,
```

Comment:
We should make `5 * 60` a `const` in `global.rs` and reuse it in `get_future_time_limit()` above.

Reply:
Sounds good; but then we should also use config::API_SECRET_FILE_NAME rather than ".api_secret" just above?

Reply:
default now defined as global::DEFAULT_FUTURE_TIME_LIMIT